### DISCUSSION TITLE
Bank Tags (tabs)

### DISCUSSION THREAD

**Discussion by ppyLEK** [runelite/runelite#16013] [Ideas]
At the moment if you play multiple accounts (main, early ironman etc) the gear difference is massive leading to massive list of tag tabs shown to the user if all of the accounts have different tag tabs.

Could this be changed so that tags and tabs are account specific?

**Comment by Nightfirecat** [comment]
#2220 is basically the same as this request. It's not a simple feature for us to add, which is why it hasn't happened yet.

The [Plugin presets hub plugin](https://runelite.net/plugin-hub/show/plugin-presets) might be able to do this?

**Reply by antero111** [reply]
With Plugin Presets you can modify the `banktags.tagtabs` setting (by adding it as a custom setting) to change bank tag visiblity across presets. You could then only show your mains bank tags on your main preset and then show your irons tags on iron presets etc. You don't modify any of the tag settings, only the visible tags on the banks tag list. Its somewhat tedious process but its possible. 

@ppyLEK please create an issue on the Plugin Presets repo if you need more help.
